### PR TITLE
GSNV2 update

### PR DESCRIPTION
**Pull request by nuevoalex**
@Mergifyio backport dev

### CODE REVIEW

**Comment by pgebheim** [comment]
@mergifyio backport dev

File: packages/augur-core/src/contracts/AugurWalletRegistryV2.sol [hunk 1]
```diff
@@ -51,7 +51,7 @@ contract AugurWalletRegistryV2 is Initializable, BaseRelayRecipient, Forwarder {
     uint256 private constant POST_RELAY_GAS_COST = 200000;
 
     // Gas stipends for acceptRelayedCall, preRelayedCall and postRelayedCall
-    uint256 constant private ACCEPT_RELAYED_CALL_GAS_LIMIT = 50000;
+    uint256 constant private ACCEPT_RELAYED_CALL_GAS_LIMIT = 75000;
```
Comment (pgebheim) [inline packages/augur-core/src/contracts/AugurWalletRegistryV2.sol:54]: tldr on why this needs to be bumped?

**Review by pgebheim — APPROVED** [review]

File: packages/augur-core/src/contracts/AugurWalletRegistryV2.sol (hunk 1 above)
Comment (nuevoalex) [reply packages/augur-core/src/contracts/AugurWalletRegistryV2.sol:54]: The signature validation was removed as part of the standard relayHub validation and made incumbent upon the recipient to validate in this call, so we have to do more in it.

**Comment by pgebheim** [comment]
@mergigyio rebase

**Comment by pgebheim** [comment]
@Mergifyio rebase

**Comment by pgebheim** [comment]
@mergifyio update

**Pull request merged**